sui_types/src/committee.rs

```diff
@@ -8,27 +8,36 @@ use rand::distributions::{Distribution, Uniform};
 use rand::rngs::OsRng;
 use std::collections::{BTreeMap, HashMap};
 
+pub type EpochId = u64;
+
 #[derive(Eq, PartialEq, Clone, Debug)]
 pub struct Committee {
+    pub epoch: EpochId,
     pub voting_rights: BTreeMap<AuthorityName, usize>,
     pub total_votes: usize,
+    // Note: this is a derived structure, no need to store.
```

Comment:
We could you user `#[serde(skip)]` for transient fields (in a future PR).

Reply:
Yes, we should -- but right now this stuct is not Serde Serizalize / Deserialize hence why we just have a comment.